netbench/netbench-driver/src/bin/netbench-driver-s2n-quic-server.rs

```diff
@@ -79,9 +79,7 @@ impl Server {
         let mut io_builder =
             io::Default::builder().with_receive_address((self.opts.ip, self.opts.port).into())?;
 
-        if self.disable_gso {
-            io_builder = io_builder.with_gso_disabled()?;
-        }
+        io_builder = io_builder.with_gso_disabled()?;
```

Comment:
why this change?

Reply:
My containers need GSO to be disabled to work with ECS, but I shouldn't have committed that, I will remove it.